akka-http-core/src/main/resources/reference.conf

```
@@ -460,6 +460,26 @@ akka.http {
     client = {
       # no overrides by default, see `akka.http.client` for default values
     }
+
+    # Add client specific overrides to host-connection-pool. Allows exact and prefixed urls
+    # eg:
+    # per-host-override = [
+    # {
+    #   "akka.io" = { # can use same things as in global `host-connection-pool` section
+    #     max-connections = 10
+    #   }
+    # },
```

An alternative encoding could be to put the pattern into a field to remove one level of nesting.

```suggestion
    # {
    #   host-pattern = "akka.io"
    #   max-connections = 10
    # },
```

---

Yes, I think that's better. Changing.